Configs/App-Debug.xcconfig

```diff
@@ -3,3 +3,9 @@
 
 #include "Module-Debug.xcconfig"
 #include "App-Shared.xcconfig"
+
+// Space-separated list of preprocessor macros of the form `foo` or `foo=bar`
+GCC_PREPROCESSOR_DEFINITIONS = $(inherited) DEBUG=1 DD_LOG_LEVEL=DDLogLevelAll
```

Comment:
just to make it clear:
`Module-Debug.xcconfig` sets `GCC_PREPROCESSOR_DEFINITIONS = $(inherited) DEBUG=1` but in this case we are shadowing that setting, so that's why I made it explicit here.